RecoTracker/ConversionSeedGenerators/python/PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py

```diff
@@ -23,7 +23,7 @@
                                                   ClusterCheckPSet = cms.PSet(
                                                       PixelClusterCollectionLabel = cms.InputTag("siPixelClusters"),
                                                       MaxNumberOfCosmicClusters = cms.uint32(400000),
-                                                      doClusterCheck = cms.bool(True),
+                                                      doClusterCheck = cms.bool(False),
```

Comment:
I think this should be controlled with `phase1Pixel` era as we do for the iterative tracking seeding in
https://github.com/cms-sw/cmssw/blob/master/RecoTracker/TkSeedGenerator/python/trackerClusterCheck_cfi.py
(and maybe eventually convert to reading the same boolean from event?).